doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1716,6 +1773,21 @@ interpretation, e.g.:
 
 are all equivalent.
 
+.. example::
+
+   .. coqtop:: reset
+
+      From Coq Require Import ssreflect.
+      Set Implicit Arguments.
+      Unset Strict Implicit.
+      Unset Printing Implicit Defensive.
+
+   .. coqtop:: all
+
+      Record r := { a : nat; b := (a, 3); _ : bool; }.
+
+      Lemma test : r -> True.
+      Proof. move => [^ x ].
```

Comment:
I would prefer if this example was closer to the description.

Reply:
The doc of i_block is not closer to the example